storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/delete/DeleteResult.java

```diff
@@ -2,28 +2,81 @@
 
 import android.support.annotation.NonNull;
 
-import java.util.Collections;
-import java.util.Set;
+import static com.pushtorefresh.storio.util.Checks.checkNotNull;
 
+/**
+ * Immutable container for result of Delete Operation
+ * <p>
+ * Instances of this class are Immutable
+ */
 public class DeleteResult {
 
-    private final int numberOfDeletedRows;
-    private final @NonNull Set<String> affectedTables;
+    private final int numberOfRowsDeleted;
 
-    private DeleteResult(int numberOfDeletedRows, @NonNull Set<String> affectedTables) {
-        this.numberOfDeletedRows = numberOfDeletedRows;
-        this.affectedTables = Collections.unmodifiableSet(affectedTables);
+    @NonNull
+    private final String affectedTable;
+
+    private DeleteResult(int numberOfRowsDeleted, @NonNull String affectedTable) {
+        checkNotNull(affectedTable, "Please specify affected table");
```

Comment:
Annotation does not guarantee anything, unfortunately. I am not happy to use `checkNotNull()` because of little performance downgrade, but I think it's better to fail fast than allowing to use object with corrupted state.